src/components/layout/nav.js

```diff
@@ -786,6 +787,7 @@ const StyledNavRight = styled(NavRight)`
     > a:last-child {
         pointer-events: ${(props) => (props.move ? 'visible' : 'none')};
         cursor: ${(props) => (props.move ? 'pointer' : 'default')};
+        opacity: ${(props) => (props.move ? 1 : 0)};
```

Comment:
If the reason of this opacity is to hide, perhaps we can use visibility: visible || hidden